frontend/src/container/NewWidget/LeftContainer/QuerySection/index.tsx

```diff
@@ -223,6 +211,16 @@ function QuerySection({
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [handleRunQuery]);
 
+	useEffect(() => {
+		// switch to query builder if query type is not supported
+		if (
+			selectedGraph === PANEL_TYPES.TABLE &&
```

Comment:
qq : this is to handle existing promQL query dashboards ?

Reply:
yes, like someone is coming from promql (in lets say time-series) to a table panel, then we will smartly switch to query-builder